### ISSUE TITLE
replace .glade files

### ISSUE THREAD

**Issue by cmaeder** [spechub/Hets#1506]
The last version of http://hackage.haskell.org/package/glade is `0.12.5` that does not work with `ghc-7.10.1`. In order to continue using our GTK interface we must port our .glade files under
https://github.com/spechub/Hets/tree/master/GUI/Glade to .gtk files. (I tried this once without success, though.)

**Comment by jelmd** [comment]
https://github.com/mpickering/glade/commit/82dc8f148bf39bc18fe224c0b27d8eeeb0c08342 ???

**Comment by cmaeder** [comment]
the PR did not compile https://travis-ci.org/gtk2hs/glade/builds/57084714

**Comment by cmaeder** [comment]
Well, it did not compile with ghc-7.6.3 (and was not tested against ghc-7.10)

**Comment by cmaeder** [comment]
I made glade installable using https://github.com/cmaeder/glade but more changes (with conditional compilation) need to be made to hets:

```
[655 of 767] Compiling GUI.GtkProverGUI ( GUI/GtkProverGUI.hs, GUI/GtkProverGUI.o )

GUI/GtkProverGUI.hs:250:49:
    Couldn't match type ‘[Char]’ with ‘Data.Text.Internal.Text’
    Expected type: [ComboBoxText]
      Actual type: [String]
    In the second argument of ‘($)’, namely ‘expand f’
    In a stmt of a 'do' block:
      mapM_ (comboBoxAppendText cbComorphism) $ expand f
make: *** [hets] Fehler 1
```

**Comment by jelmd** [comment]
What about this?
[gtk.txt](https://github.com/spechub/Hets/files/54453/gtk.txt)

**Comment by cmaeder** [comment]
Thanks for the patch. I still need to make a patch that works for ghc-7.6 (and old glade bundled with Ubuntu) and ghc-7.10.

**Comment by cmaeder** [comment]
On wily the package libghc-glade-dev is missing. Therefore more work is needed to make hets for wily.
(Maybe one should try to convert .glade files to .gtk files, again.) On the other hand our current use of dialog files is stupid as they are recreated at runtime from internal strings (due to the lack of proper resource bundling). Therefore creating the dialogs by code directly would be another option.

**Comment by jelmd** [comment]
Found https://launchpad.net/ubuntu/+source/haskell-glade/0.12.1-4build2 - not sure, how the submission stuff for ubuntu works. However, if in doubt, making a own package may work?

**Comment by cmaeder** [comment]
Yes, putting libghc-glade-dev into our ppa is an (may the simplest) option, too.
